src/python/gem5/components/boards/kernel_disk_workload.py

```diff
@@ -39,6 +39,7 @@
 from pathlib import Path
 
 import m5
+from m5.util import inform
```

Comment:
You import this bug never use it. I did ask for an inform to be thrown when setting the kernel arguments, but this is never done. Did you forget to commit it?